ui/src/app/base/components/FormikFormContent/FormikFormContent.test.tsx

```diff
@@ -268,4 +269,56 @@ describe("FormikFormContent", () => {
     wrapper.update();
     expect(wrapper.find("input[name='val1']").props().value).toBe("initial");
   });
+
+  it("runs onSuccess function if successfully saved with no errors", async () => {
```

Comment:
@sowasred2012 @bethcollins92 Instead of writing a test in the zone list header, I've written a much simpler one here that just asserts that the `onSuccess` function is called when expected.

The test you had started writing in the zone list header extended a bit beyond the boundaries of the component itself (i.e. mocking redux actions) and is really tricky to get right. I think in general, at least for this sprint, we can just write tests that assert that given some inputs (store/props) and optionally some internal actions (click a button), we can expect some outputs (action is dispatched from the component/form is closed).